specs/sql/README.md

```diff
@@ -335,18 +335,133 @@ contain only column names. An error is raised if more than one
 `PRIMARY KEY` clause appears in a `CREATE TABLE` statement. The
 `PRIMARY KEY` is optional.
 
+If a table has a single column primary key and the declared type of that
+column is `INTEGER`, then the column is known as an
+`INTEGER PRIMARY KEY`. See below for a description of the special
+properties and behaviors associated with an
+[`INTEGER PRIMARY KEY`](#integer-primary-key).
+
 Each row in a table with a primary key must have a unique combination of
 values in its primary key columns. If an `INSERT` or `UPDATE` statement
 attempts to modify the table content so that two or more rows have
 identical primary key values, that is a constraint violation. Related,
 the SQL standard is that a `PRIMARY KEY` should always be `NOT NULL`, so
 Tableland enforces this constraint.
 
-> 🚧 **Feature At Risk**: It is not currently possible to `ALTER TABLE`
-> after it has been created. As such, table structure in Tableland is
-> considered immutable. The Tableland core development team is currently
-> evaluating whether to allow `ALTER TABLE` only in the case of *adding
-> new columns*.
+#### Integer Primary Key
+
+All rows within Tableland tables have a 64-bit signed integer key that
+uniquely identifies the row within its table. This integer is usually
+called the `ROWID`. The `ROWID` value can be accessed using one of the
+special case-independent names `"rowid"`, `"oid"`, or `"_rowid_"` in
+place of a column name. If a table contains a user defined column named
```

Comment:
> If a table contains a user defined column named
`"rowid"`, `"oid"` or `"_rowid_"`, then that name always refers the
explicitly declared column and cannot be used to retrieve the integer
`ROWID` value.

In my implementation, I'm blocking `"rowid"`, `"oid"`, or `"_rowid_"` as possible identifiers for columns in a `CREATE TABLE` statement. The reason for that is that I have to block those columns on `INSERT` and `UPDATE` to follow the spec, and I don't see a reason for allowing those columns on `CREATE` since they will not be able to be touched.

cc @jsign

Reply:
Agree. Also, it makes the spec simpler by removing these border cases with weird aliasing rules.

Reply:
Perfect, sounds good to me!